### DISCUSSION TITLE
colab v2 Training error

### DISCUSSION THREAD

**Discussion by gyuchulm** [acids-ircam/RAVE#189] [Q&A]
![image](https://user-images.githubusercontent.com/9707888/217960695-98ffb7ad-026f-4d3a-ade8-26f9b67eb103.png)

Do I miss something at this stage? 

Save dir is not referred to in this stage; I didn’t edit any code except deleting the previous dataset. 

Does anyone have the same issues? 

```
!/content/miniconda/bin/rave train --config $architecture --config $regularization --db_path $preprocessed_dataset --name $name --ckpt $resume
```
The only thing I can check the different with the original code.

**Comment by devstermarts** [comment]
Had the same problem. The resume variable is not set in this section of the notebook.  
Deleting `--ckpt $resume` did the trick for me. 
It's set in the respective line in the following section "Resume training" where it works perfectly fine.

**Comment by gyuchulm** [comment]
It works, and there is no "default. gin" file in the rave config. 
I deleted the regulation and ran it, then it works

**Comment by twobob** [comment]
Example, working, training section
![image](https://user-images.githubusercontent.com/915232/218337319-7fae4693-d8b2-4266-bc14-06439f5101ab.png)

**Reply by gyuchulm** [reply]
Have you tried with the default config??
